src/app/Openshift/Openshift.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { PageSection, Title }  from '@patternfly/react-core';
+import { Fetchdata } from '@app/fetchdata/fetchdata';
```

Comment:
Resolve merge conflict here.

Reply:
Conflict resolved , import required to display the data on OpenShift tab, hence added .